app/services/gtfs_graph.rb

```diff
@@ -23,8 +23,9 @@ def create_change_osr(import_level=0)
     load_tl_stops
     load_tl_routes
     operators = load_tl_operators
-    routes = operators.map { |operator| operator.serves }.reduce(Set.new, :+)
-    stops = routes.map { |route| route.serves }.reduce(Set.new, :+)
+    routes = operators.map { |operator| operator.serves }.reduce(Set.new, :+).map { |i| find_by_onestop_id(i) }
```

Comment:
Line is too long. [111/80]<br>Pass `&:serves` as an argument to `map` instead of a block.